src/frontend/src/components/public/PIAFormTabs/intake/printPreview/intakedetails.tsx

```diff
@@ -4,6 +4,7 @@ import { useState } from 'react';
 import MDEditor from '@uiw/react-md-editor';
 
 const IntakeDetails = (pia: IPiaForm) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const [piaMinistryFullName, setPiaMinistryFullName] = useState(
```

Comment:
We should investigate why some of these useStates are only being used for either the setter or the getter